src/utils/encoders.ts

```diff
@@ -64,6 +64,29 @@ export const encodeOrders = ([order0, order1]: [DecodedOrder, DecodedOrder]): [
   return [encodeOrder(order0), encodeOrder(order1)];
 };
 
+export const isOrderEncodable = (order: DecodedOrder) => {
+  try {
+    encodeOrder(order);
+    return true;
+  } catch (e) {
+    return false;
+  }
+};
+
+/**
+ * Checks if the rates are equal after scaling them
+ * @param {string} x - the first rate
+ * @param {string} y - the second rate
+ * @returns {boolean} - true if the rates are equal after scaling, false otherwise
+ */
+export const scaleRatesAndCheckIsEqual = (x: string, y: string) => {
+  const xDec = new Decimal(x);
+  const yDec = new Decimal(y);
+  const xScaled = DecToBn(encodeRate(xDec));
```

Comment:
Why do you transform it into big number?

Reply:
I intentionally stick with the exact same process that a rate goes through during encodeOrder